tests/system/source/main.d

```diff
@@ -127,7 +140,7 @@ private void assertBlockHeight (const string[] addresses, ulong height)
             {
                 writefln("%s Client #%s is at block height %s", PREFIX, idx, getHeight);
                 const blocks = client.getBlocksFrom(0, 42);
-                writefln("%s Client #%s has blocks:\n%s", PREFIX, idx, blocks.map!prettify);
+                writefln("%s Client #%s has blocks:\n%s", PREFIX, idx, prettify(blocks));
```

Comment:
This should be on another commit because it's a style change.

Reply:
I will split the changes into two commits.

Reply:
Updated